docs/tutorials/fixer.md

```diff
@@ -0,0 +1,120 @@
+# Prowler Fixer
+Prowler allows you to fix some of the failed findings it identifies. You can use the `--fixer` flag to run the fixes that are available for the checks that failed.
+
+```sh
+prowler <provider> -c <check_to_fix_1> <check_to_fix_2> ... --fixer
+```
+
+<img src="../img/fixer.png">
+
+???+ note
+    You can see all the available fixes for each provider with the `--list-fixers` flag.
+
+    ```sh
+    prowler <provider> --list-fixer
+    ```
+
+## Writing a Fixer
+To write a fixer, you need to create a file called `<check_id>_fixer.py` inside the check folder, with a function called `fixer` that receives either the region or the resource to be fixed as a parameter, and returns a boolean value indicating if the fix was successful or not.
```

Comment:
What if the fixer needs the `region` and the `resource_id`?

Reply:
It is not contemplated, but good point...